client/packages/common/src/ui/components/inputs/Filters/DateFilter.tsx

```diff
@@ -86,22 +85,40 @@ const getDateFromUrl = (query: string, range: RangeOption | undefined) => {
   return DateUtils.getDateOrNull(value);
 };
 
-const getRangeBoundary = (
+export const getMinMaxDates = (
   query: string | RangeObject<string>,
   range: RangeOption | undefined,
-  limit: Date | string | undefined
+  min: Date | string | undefined,
+  max: Date | string | undefined
 ) => {
-  const limitDate = DateUtils.getNaiveDate(limit);
-  if (typeof query !== 'object' || !range) return limitDate || undefined;
+  const minDate = DateUtils.getDateOrNull(min);
+  const maxDate = DateUtils.getDateOrNull(max);
+
+  if (typeof query !== 'object' || !range) {
+    return {
+      minDate: minDate || undefined,
+      maxDate: maxDate || undefined,
+    };
+  }
+
   const { from, to } = query as RangeObject<string>;
```

Comment:
but why is it a string ? in definition ?

Reply:
in method definition ('string' | RangeObject)

Reply:
I thinkk because query comes from URL, and URL query string can be anything... tbh didn't think too deep about existing types was just trying to get the logic going 🙈

Reply:
That's fine, thanks